cumulusci/tasks/push/tasks.py

```diff
@@ -214,7 +215,12 @@ class SchedulePushOrgList(BaseSalesforcePushTask):
         },
         "version": {
             "description": "The managed package version to push",
-            "required": True,
+        },
+        "version_id": {
+            "description": "The MetadataPackageVersionId (ID prefix `04t`) to push",
+        },
+        "metadata_package_id": {
+            "description": "The MetadataPackageId (ID prefix `033`) to push.",
```

Comment:
Is it worth adding validation to make sure this and namespace aren't specified at the same time?

Reply:
@davisagli added line 284 below